_posts/2025-10-21-flashinfer-bench.md

```
@@ -0,0 +1,202 @@
+    A: torch.Tensor,
+    B: torch.Tensor,
+) -> torch.Tensor:
+    # Fallback Implementation
+    return torch.matmul(A, B)
+```
+
+It takes a lambda function that maps the inputs to a specific kernel definition name, then replaces the original computation function. The replaced function filters the best-performing kernel solution from the database based on the inputs, executes it, and returns its result. If no matching kernel solution is found, the original function runs as a fallback. 
+
+By using `apply()`, we can replace a large number of operators in FlashInfer. If you want to replace operators from your own implementations or other libraries, you can also easily achieve this with `apply()`.
+
+##FlashInfer-Bench Leaderboard — The Arena for LLM Kernel Optimization
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)